cmd/collectors/ems/ems_test.go

```diff
@@ -82,6 +85,10 @@ func (e *Ems) testBookendIssuingEms(t *testing.T, path string) {
 	// Polling ems collector to handle results
 	if _, emsCount, _ := e.HandleResults(results, e.emsProp); emsCount == 0 {
```

Comment:
minor but this can be simplified to
```go
if _, emsCount, _ := e.HandleResults(results, e.emsProp); emsCount != expectedInstanceLabelCount {
    t.Errorf("Instance labels count mismatch detected. Got numLabels: %d want numLabels: %d", emsCount, expectedInstanceLabelCount)
}
```

Reply:
Done